bitnami/clickhouse/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: clickhouse
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/clickhouse
   - https://github.com/ClickHouse/ClickHouse
-version: 2.1.1
+version: 2.1.2
```

Comment:
Could you please bump a minor version? (2.2.0)

I don't think a major would be needed as no backward compatibility is broken, but modifying the number of replicas shouldn't be included in a patch version.

Users may be surprised if upgrading from 2.1.1 to 2.1.2 changed the number of pods in their deployment.

Reply:
ok, i will tag 2.2.0